mesheryctl/internal/cli/root/relationships/relationship.go

```diff
@@ -0,0 +1,159 @@
+			return utils.ErrInvalidArgument(errors.New("missing required argument: [model-name]. " + errMsg))
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
+			return errors.New(utils.RelationshipsError(fmt.Sprintf("'%s' is an invalid subcommand. Please provide required options from [view]. Use 'mesheryctl exp relationships --help' to display usage guide.\n", args[0]), "relationship"))
+		}
+		_, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return utils.ErrLoadConfig(err)
+		}
+
+		err = cmd.Usage()
+		if err != nil {
+			utils.Log.Error(err)
+			return nil
+		}
+		return nil
+	},
+}
+
+func init() {
+	ViewRelationshipsCmd.Flags().StringVarP(&outFormatFlag, "output-format", "o", "yaml", "(optional) format to display in [json| yaml]")
+
+	RelationshipCmd.AddCommand(availableSubcommands...)
+}
+
+// selectModelPrompt lets user to select a relation if relations are more than one
+func selectRelationshipPrompt(relationship []v1alpha1.RelationshipDefinition) v1alpha1.RelationshipDefinition {
+	relationshipArray := []v1alpha1.RelationshipDefinition{}
+	relationshipNames := []string{}
+
+	relationshipArray = append(relationshipArray, relationship...)
+
+	for _, relationship := range relationshipArray {
+		// here display Kind and EvaluationQuery as relationship name
+		relationshipName := fmt.Sprintf("kind: %s, EvaluationPolicy: %s, SubType: %s", relationship.Kind, relationship.EvaluationQuery, relationship.SubType)
+		relationshipNames = append(relationshipNames, relationshipName)
+	}
+
+	prompt := promptui.Select{
+		Label: "Select a relationship:",
+		Items: relationshipNames,
+	}
+
+	for {
+		i, _, err := prompt.Run()
+		if err != nil {
+			continue
+		}
+
+		return relationshipArray[i]
+	}
+}
+
+func outputRelationshipJson(model v1alpha1.RelationshipDefinition) error {
+	if err := prettifyRelationshipJson(model); err != nil {
+		// if prettifyJson return error, marshal output in conventional way using json.MarshalIndent
+		// but it doesn't convert unicode to its corresponding HTML string (it is default behavior)
+		// e.g unicode representation of '&' will be printed as '\u0026'
+		if output, err := json.MarshalIndent(model, "", "  "); err != nil {
+			return errors.Wrap(err, "failed to format output in JSON")
+		} else {
+			fmt.Print(string(output))
+		}
+	}
+	return nil
+}
+
+// prettifyJson takes a v1alpha1.RelationshipDefinition struct as input, marshals it into a nicely formatted JSON representation,
+// and prints it to standard output with proper indentation and without escaping HTML entities.
+func prettifyRelationshipJson(relationship v1alpha1.RelationshipDefinition) error {
+	// Create a new JSON encoder that writes to the standard output (os.Stdout).
+	enc := json.NewEncoder(os.Stdout)
```

Comment:
If I am not wrong we are using similar logic for components/models, so it is better to make it generic. 
Also accept the writer as argument, instead of just specifying it as `os.Stdout` so that function is reusable, even if we want to write to file or response....